alpha/lib/model/kuser.php

```diff
@@ -86,7 +86,9 @@ public function preSave(PropelPDO $con = null)
 					}
 				}
 			}
-			
+
+			$this->setUpdatedAt(time());
```

Comment:
move this after all the if validations have passed.
The next if statement may throw an exception so there is not point to set updated at before it passed.